src/docs/product/user-feedback/index.mdx

```diff
@@ -24,10 +24,4 @@ User feedback is only applicable for [error issues](/product/issues/issue-detail
 
 Sentry provides two ways to collect user feedback, an API or an embeddable widget, which you can [configure](/platform-redirect/?next=/enriching-events/user-feedback/) to collect feedback when a user experiences an error.
```

Comment:
```suggestion

There are two ways to collect user feedback in Sentry: via an API, or by using an embeddable widget, which you can [configure](/platform-redirect/?next=/enriching-events/user-feedback/) to collect feedback when a user experiences an error. For JVM-related use of Kotlin, check out our [Java](/platforms/java/enriching-events/user-feedback/) or [Android](/platforms/android/enriching-events/user-feedback/) SDKs.
```

Reply:
Since we're removing the unsupported list, we should also remove the first part of the paragraph below it:

"Clicking the "configure" link above for these SDKs may result in a "Page Not Found" error."